hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/OpenKeyCleanupService.java

```diff
@@ -96,6 +98,17 @@ public OpenKeyCleanupService(long interval, TimeUnit unit, long timeout,
         TimeUnit.MILLISECONDS);
     this.expireThreshold = Duration.ofMillis(expireMillis);
 
+    long leaseHardMillis = conf.getTimeDuration(OMConfigKeys.OZONE_OM_LEASE_HARD_LIMIT,
+        OMConfigKeys.OZONE_OM_LEASE_HARD_LIMIT_DEFAULT, TimeUnit.MILLISECONDS);
+    long leaseSoftMillis = conf.getTimeDuration(OzoneConfigKeys.OZONE_OM_LEASE_SOFT_LIMIT,
+        OzoneConfigKeys.OZONE_OM_LEASE_SOFT_LIMIT_DEFAULT, TimeUnit.MILLISECONDS);
+
+    if (leaseHardMillis < leaseSoftMillis) {
+      this.leaseThreshold = Duration.ofMillis(leaseSoftMillis);
```

Comment:
Can you log a warning message here?

Reply:
Added

Reply:
Changed to throw `IllegalArgumentException`.